.travis.yml

```diff
@@ -25,10 +26,8 @@ notifications:
   email:
     on_success: never
 
-sudo: false
-
 script:
-  - ./gradlew clean test
+  - ./gradlew clean cobertura coveralls
```

Comment:
The `cobertura` command will both run tests and generate the coverage reports. In theory, it should be possible to separate the two with the `test` and `coberturaReport` commands, but I couldn't make the latter work for some reason.